backend/src/hooks/able-to-write-interview-notes/able-to-write-interview-notes.hook.ts

```diff
@@ -0,0 +1,42 @@
+import { FastifyRequest } from 'fastify';
+
+import {
+  ExceptionMessage,
+  HttpCode,
+  PermissionKey,
+} from '~/common/enums/enums';
+import { PermissionsError } from '~/exceptions/exceptions';
+import { checkHasPermission } from '~/helpers/helpers';
+
+const checkIsAbleToWriteInterviewNote =
```

Comment:
It looks similar to `checkHasPermissions` hook but you hardcoded permissionKeys 🤔 
Won't it be better to use existing common hook? Or there is any hidden concern for it?

Reply:
Yep, if I just use hook we have it will check both permissions, but here user needs MANAGE_INTERVIEW or MANAGE_INTERVIEWS

Reply:
you can modify `checkHasPermissions` to provide a generic `type: 'every' | 'oneOf'` and change a logic